README.md

```diff
@@ -316,17 +321,14 @@ $ kubectl apply -f demo-storageclass-gold-pvc.yaml
 storageclass.storage.k8s.io/gold created
 ```
 
-
-
-
 <br/>
 <br/>
 <br/>
 
 
 ## Driver Usage
 > **Note**: For further usage details, refer to https://github.com/IBM/ibm-block-csi-driver. 
->          In addition, full documentation can be found on the [IBM knowledge center](https://www.ibm.com/support/knowledgecenter/SSRQ8T).
+>          In addition, full documentation can be found on the [IBM knowledge center](https://www.ibm.com/docs/en/stg-block-csi-driver).
```

Comment:
In addition, for full product information, see [IBM block storage CSI driver documentation](https://www.ibm.com/docs/en/stg-block-csi-driver).

Reply:
done